.github/workflows/deploy_netlify.yml

```diff
@@ -2,12 +2,9 @@ name: Netlify Deployments
 
 on:
   workflow_run:
-    workflows: ["Unlock Test Suite"]
+    workflows: [Unlock Test Suite]
     types:
       - completed
-    branches:
-      - master
-      - production
 
```

Comment:
Would removing this mean it will run on all branches by default? if so, LGTM

Reply:
yes, thats the idea